### PR TITLE
TASK-602

### PR DESCRIPTION
**Pull request by pfurio**
TASK-602

### CODE REVIEW

**Comment by pfurio** [comment]
Task linked: [TASK-602 Allow searching by  genes.gene_name in end point panels/search  parameter genes](https://app.clickup.com/t/36631768/TASK-602)

File: opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_2_1/catalog/AddPanelGeneNameIndex.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        domain = Migration.MigrationDomain.CATALOG,
+        date = 20220406)
+public class AddPanelGeneNameIndex extends MigrationTool {
+
+    @Override
+    protected void run() throws Exception {
+        Document newIndex = new Document()
+                .append("genes.name", 1)
+                .append("studyUid", 1);
+        createIndex(MongoDBAdaptorFactory.PANEL_COLLECTION, newIndex);
```
Comment (j-coll) [inline opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_2_1/catalog/AddPanelGeneNameIndex.java:20]: wouldn't be better to call `catalogManager.installIndexes(token)` ?

Comment (pfurio) [reply opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_2_1/catalog/AddPanelGeneNameIndex.java:20]: It's a possibility, although that will check all indexes. This migration is simply to add that specific index, not to review all of them.

Comment (j-coll) [reply opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_2_1/catalog/AddPanelGeneNameIndex.java:20]: that could be a "default" migration step on each release. "EnsureIndexesExist"

Comment (pfurio) [reply opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_2_1/catalog/AddPanelGeneNameIndex.java:20]: Agree. Actually I've done that in previous releases when indexes have changed considerably.

**Pull request merged**